packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects.
+        */
+        'func-names': ['error', 'as-needed'],
+        indent: ['error', 4],
+        'linebreak-style': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
+            where it is appropriate to disable this rule for older projects or with a comment.
+            Including one class per file is a best practice and recommended by the Angular style
+            guide. However, migrating older projects may not be trivial, and there may be
+            exceptions for organization.
```

Comment:
I propose changing "there may be exceptions for organization" to something more specific. We could borrow the language from our C# conventions: "Exceptions may be made to the one "main" type per file rule for public/internal types that are only used as part of the interface to the main type and no other types."

Reply:
Yes, your suggestion is better.

Reply:
@jattasNI Is that comment supposed to be "private/internal types"?

Reply:
I think in the C# rules they're ok with even having multiple public types in a file if they are tightly coupled (an enum and the API that uses it or a factory and the type it produces, for example). 

If you think it's confusing, I would be ok with making it more vague and just removing "public/internal" in our case or adopting another suggestion if you have one.

Reply:
I also find the original wording confusing and like Jesse's suggestion.